shifter/shifter_files/cron.py

```diff
@@ -0,0 +1,7 @@
+from django.db.models.functions import Now
+from .models import FileUpload
+
+
+def delete_expired_files():
+    FileUpload.objects.filter(expiry_datetime__lte=Now()).delete()
+    print("deleting")
```

Comment:
Can you remove the debug print message please

Reply:
I didn't finish yet...
And I'm keeping tests in mind. 
I just the changes have been added automatically. SORRY!
working on it.

Reply:
Ah my apologies! Shoot me a message once it's done!